services/app-web/src/pages/StateSubmission/SubmissionType/SubmissionType.tsx

```diff
@@ -239,13 +251,50 @@ export const SubmissionType = ({
                                     </Alert>
                                 ))}
                             <span>All fields are required</span>
-                            <FieldDropdown
-                                id="programID"
-                                name="programID"
-                                label="Program"
-                                showError={showFieldErrors(errors.programID)}
-                                options={programOptions}
-                            />
+                            <FormGroup
+                                error={showFieldErrors(errors.programIDs)}
+                            >
+                                <Label htmlFor="programIDs">Program(s)</Label>
+                                {showFieldErrors(errors.programIDs) && (
+                                    <ErrorMessage>
+                                        {errors.programIDs}
+                                    </ErrorMessage>
+                                )}
+                                <Field name="programIDs">
+                                    {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
+                                    {/* @ts-ignore */}
+                                    {({ field, form }) => (
+                                        <Select
```

Comment:
OK now I think I get the AriaLive component above. It seems like we aren't wrapping the Select from react-select in anything, so we don't really have our own component here, so the component in /components is really a demo for storybook. 

I think (and @haworku please weigh in) that our /components should be stuff that we would import and use elsewhere in the app. 

It would make more sense for me for either 

a) us to create a MultiSelect wrapping component like we have with a lot of the USWDS stuff. 

or 

b) if we don't need that, then to drop that component all together. 

The Color select demo component code would make more sense to me directly in storybook. 

Curious to hear what you think

Reply:
Since the code will be preserved in git, maybe I'll drop this altogether for now, and we can reference it when we create an actual component/storybook implementation.  @haworku and I did talk a bit yesterday about the need for a wrapped multiselect component...